tools/bin/mbedtls-rewrite-branch-style

```diff
@@ -477,7 +496,23 @@ class CodeStyleBranchRewriter(MbedTLSBranchRewriter):
         # cherry-picked commit, but might mess up modified files. Then
         # we override the content of modified files, and finally we restyle
         # files that need restyling.
-        self.run_git(['cherry-pick', '-Xtheirs', '--allow-empty', new_commit])
+        if deleted_styled_files:
+            # There are some deleted files, so expect cherry-pick to return
+            # with exit status 1.
+            try:
+                self.run_git(['cherry-pick', '-Xtheirs', '--allow-empty', new_commit])
+            except subprocess.CalledProcessError as c:
+                if c.returncode != 1:
+                    raise c
+            # Resolve conflicts by removing any deleted files then continue.
+            self.run_git(['rm'] + deleted_styled_files)
+            # Continue the cherry-pick with the editor set to the 'true'
+            # command which returns success immediately. In other words,
+            # don't open an editor.
+            self.run_git(['cherry-pick', '--continue'],
+                         env={**os.environ, 'GIT_EDITOR':'true'})
```

Comment:
I think `GIT_EDITOR=true` won't work on Windows. Unfortunately `git cherry-pick` lacks a `--no-edit` option except in very recent git versions. I think `git commit --no-edit --allow-empty` will work here, except it adds a `# Conflicts:` block to the log message (we'd have to delete `MERGE_MSG` to avoid that).